src/ui/page/channels/view.jsx

```diff
@@ -0,0 +1,48 @@
+// @flow
+import React, { useEffect } from 'react';
+import ClaimList from 'component/claimList';
+import Page from 'component/page';
+import Button from 'component/button';
+import YoutubeTransferStatus from 'component/youtubeTransferStatus';
+
+type Props = {
+  channels: Array<ChannelClaim>,
+  fetchChannelListMine: () => void,
+  fetchingChannels: boolean,
+  youtubeChannels: ?Array<any>,
+};
+
+export default function ChannelsPage(props: Props) {
+  const { channels, fetchChannelListMine, fetchingChannels, youtubeChannels } = props;
+  const hasYoutubeChannels = youtubeChannels && Boolean(youtubeChannels.length);
+
+  useEffect(() => {
+    fetchChannelListMine();
+  }, [fetchChannelListMine]);
+
+  return (
+    <Page>
+      {hasYoutubeChannels && <YoutubeTransferStatus />}
+
+      {channels && channels.length ? (
+        <div className="card">
+          <ClaimList
+            header={__('Your Channels')}
+            loading={fetchingChannels}
+            uris={channels.map(channel => channel.permanent_url)}
+          />
+        </div>
+      ) : (
+        <section className="main--empty">
+          <div className=" section--small">
+            <h2 className="section__title--large">{__('No Channels Created Yet')}</h2>
```

Comment:
stylistically I think empty states could still use work

_because_ the state is empty, anything being displayed already has a lot of emphasis

this empty state (and some others) use large bold text, I think this could be softened/reduced

I also think more horizontal space could be used before wrapping

Maybe look at https://material.io/design/communication/empty-states.html for how Material Design does it